src/main/scala/org/broadinstitute/dsde/firecloud/service/TSVFileSupport.scala

```diff
@@ -165,12 +165,18 @@ trait TSVFileSupport {
               case Success(ref) => ref
               case Failure(_) => AttributeString(value)
             }
-
         }
       }
     }
   }
 
+  def checkForJson(value: String): Attribute = {
+    Try(value.parseJson) match {
+        case Success(jsVal) => AttributeValueRawJson(value)
+        case Failure(_) => AttributeString(value)
+      }
```

Comment:
if you wanted to change as little as possible in this PR - that is, treat json objects as json but preserve existing behavior of everything else being treated as a string - you could check if the parsed `JsValue` is truly a json object:
```scala
    Try(value.parseJson) match {
      case Success(jso: JsObject) => AttributeValueRawJson(jso)
      case _ => AttributeString(value)
    }
```

Reply:
Ah thank you!  Saves me the trouble of digging into documentation to figure out what my options are for doing just that!